### PR TITLE
Add the BQ job usage tracking from LangChain

### PR DESCRIPTION
**Pull request by ashleyxuu**
- **Description:**
Add the BQ job usage tracking from LangChain

### CODE REVIEW

File: libs/community/langchain_community/vectorstores/bigquery_vector_search.py [hunk 1]
```diff
@@ -88,10 +89,15 @@ def __init__(
                 to use. Defaults to None.
         """
         try:
+            from google.api_core import client_info
             from google.cloud import bigquery
```
Comment (lkuligin) [inline libs/community/langchain_community/vectorstores/bigquery_vector_search.py:94]: can we reuse this method maybe?
https://github.com/langchain-ai/langchain/blob/37062549f9704765b147330b11988d8f5151f301/libs/partners/google-vertexai/langchain_google_vertexai/_utils.py#L60

**Review by lkuligin — CHANGES_REQUESTED** [review]

File: libs/community/langchain_community/vectorstores/bigquery_vector_search.py (hunk 1 above)
Comment (ashleyxuu) [reply libs/community/langchain_community/vectorstores/bigquery_vector_search.py:92]: Thanks. Done.

**Review by lkuligin — APPROVED** [review]

**Comment by ashleyxuu** [comment]
@baskaryan Could you help review and merge it?

File: libs/community/langchain_community/utils/google.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from typing import Any, Callable, Optional, Union
+
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain_core.language_models.llms import create_base_retry_decorator
+
+
+def create_retry_decorator(
```
Comment (efriis) [inline libs/community/langchain_community/utils/google.py:13]: Can probably remove this right? Not used?

**Review by efriis — APPROVED** [review]

**Pull request merged**